app/src/main/java/com/willowtree/vocable/MainActivity.kt

```diff
@@ -10,21 +10,24 @@ import com.willowtree.vocable.customviews.PointerListener
 import com.willowtree.vocable.customviews.PointerView
 import com.willowtree.vocable.databinding.ActivityMainBinding
 import com.willowtree.vocable.presets.PresetsFragment
+import com.willowtree.vocable.utils.VocableSharedPreferences
 import com.willowtree.vocable.utils.VocableTextToSpeech
 import io.github.inflationx.viewpump.ViewPumpContextWrapper
+import org.koin.android.ext.android.inject
 
 
 class MainActivity : BaseActivity() {
 
     private lateinit var binding: ActivityMainBinding
+    private val sharedPrefs: VocableSharedPreferences by inject()
     private val allViews = mutableListOf<View>()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
-        binding.pointerView.isVisible = BuildConfig.USE_HEAD_TRACKING
+        binding.pointerView.isVisible = BuildConfig.USE_HEAD_TRACKING && sharedPrefs.getHeadTrackingEnabled()
```

Comment:
how do we wanna handle this? maybe 
```suggestion
        if (BuildConfig.USE_HEAD_TRACKING) {
            binding.pointerView.isVisible = true
        } else {
            binding.pointerView.isVisible = sharedPrefs.getHeadTrackingEnabled()
        }
```

Reply:
Hmmm what's the use case for the config vs shared preference?

Reply:
so I think the config is for testing and the shared prefs one is the one that the user sets

Reply:
If there's no instance where we would want to be testing the user preferences then the suggestion makes perfect sense to me. I'm guessing we wouldn't be automating head tracking stuff, so this is really just a flag to turn it off?

Reply:
i think so, but let's check with @nerdsosky

Reply:
Yes this flag disables head tracking for the automation tests. It was added before I was on the project but that is what we are currently using it for at the least.